easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.0.0-foss-2020b.eb

```
@@ -0,0 +1,76 @@
+easyblock = 'PythonBundle'
+
+name = 'AlphaFold'
+version = '2.0.0'
+
+homepage = 'https://deepmind.com/research/case-studies/alphafold'
+description = "AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+builddependencies = [
+    # required for installing dm-tree
+    ('Bazel', '3.7.2'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('PyYAML', '5.3.1'),
+    ('TensorFlow', '2.5.0'),
+    ('Biopython', '1.78'),
+    ('jax', '0.2.19'),  # also provides absl-py
+]
```

```suggestion
    ('OpenMM', '7.5.1'),
]
```

Needed for `pdbfixer`.

---

It's also required by AlphaFold itself, `run_alphafold.py` requires `simtk` (provided by `OpenMM`)